dart/lib/src/propagation_context.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:meta/meta.dart';
+
+import '../sentry.dart';
+
+@internal
+class PropagationContext {
+  late SentryId traceId;
```

Comment:
Could we just assign the random Ids when the properties are declared? No need for a explicit constructor then.

```
@internal
class PropagationContext {
  SentryId traceId = SentryId.newId();
  SpanId spanId = SpanId.newId();
  
...

}
```